src/TextField/TextField.js

```diff
@@ -2,9 +2,19 @@
 
 import React, { Component, PropTypes } from 'react';
 import shallowEqual from 'recompose/shallowEqual';
+import { createStyleSheet } from 'jss-theme-reactor';
+import classNames from 'classnames';
 import { Input, InputLabel } from '../Input';
 import FormControl from '../Form/FormControl';
 
+export const styleSheet = createStyleSheet('TextField', () => {
+  return {
+    root: {
+      marginTop: 16,
```

Comment:
Actually, what the point of having a margin into a low-level component?
I think that we would much better off without. At @doctolib low-level component's margin have been quite a pain to handle.